phpcs.xml.dist

```diff
@@ -193,6 +193,22 @@
 			<property name="enableStandaloneNullTrueFalseTypeHints" value="false" /><!-- Only available in PHP 8.2+ -->
 		</properties>
 	</rule>
+	
+	<!-- 
+		Exclude REST controllers from type hint requirements.
```

Comment:
I had some issues going around in circles with the type hinting not matching core and want the linter wants, so temporarily disabled until I can look at this more.

Reply:
Tracked in https://github.com/WordPress/abilities-api/issues/13.